test/e2e.js

```diff
@@ -8,8 +8,9 @@ var clients = require('../lib');
 
 describe('restify-client tests against real web server', function () {
     it('have timings', function (done) {
-        var client = clients.createJsonClient({
-            url: 'https://netflix.com'
+        this.timeout(10000);
+        var client = clients.createStringClient({
```

Comment:
This is a somewhat unrelated change - just upping the timeout here and using String client instead of JSON.